astero/private/adipls_support.f90

```diff
@@ -682,8 +681,7 @@ subroutine read_and_store(iriche, iturpr, cgrav)
             write(*,*) 'read_and_store failed in read_fgong_file'
             call mesa_error(__FILE__,__LINE__)
          end if
-         call fgong_amdl( &
```

Comment:
This is a bug. The second input variable `nn` (intent(in)) does not have a defined value at this point in the code. It should be `nn_in`